src/modules/comment/addcomment.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable consistent-return */
+/* eslint-disable operator-linebreak */
```

Comment:
- [ ] Same here, Please remove this linters disablement line and fix all eslint issues yourself.